src/app/screens/connectors/ConnectUmbrel/index.tsx

```diff
@@ -103,6 +103,7 @@ export default function ConnectUmbrel() {
       submitLoading={loading}
       submitDisabled={formData.url === "" || formData.macaroon === ""}
       onSubmit={handleSubmit}
+      video="https://cdn.getalby-assets.com/Connector-guides/in_extension_guide_umbrel.mp4"
```

Comment:
I always like everything lowercase `[a-z0-9-_]` (this will never cause errors because of the case or some special characters)

Reply:
@MoritzKa Can you please rename the folder to `connector-guides`?